content/0.server-docs/3.job-guides/0.open-vs-whitelisted-jobs.md

```diff
@@ -26,14 +26,18 @@ A list of whitelisted jobs can be found below (This list is not exhaustive):
 ### DoJ Jobs
 - San Andreas Police Force
 - San Andreas Emergency Services
-- San Andreas Ranger Service
 - Judicial System - Lawyer/Paralegal
 
 ### DoC Jobs
-- Mechanics (Flywheels, Los Santos Customs, Benny’s Motorworks)
-- Car Dealerships (Premium Deluxe Motorsports, Valhalla)
+- Mechanics (Flywheels, Los Santos Customs, Benny’s Motorworks and Hayes)
+- Car Dealerships (Premium Deluxe Motorsports, Harmony Offroadsales and The District)
```

Comment:
remove this line and add dealerships to the mechanics line eg `Mechanics / Vehicle Dealership`